package.json

```diff
@@ -13,10 +13,11 @@
     "extends": "react-app"
   },
   "devDependencies": {
+    "react-localize": "github:eggheadio/react-localize#2.0.2",
```

Comment:
Using a fork for now because the original has a prop type error - I submitted a PR and will use original once merged:
https://github.com/sprjr/react-localize/issues/32